core/vdbe/sorter.rs

```diff
@@ -87,4 +87,8 @@ impl Cursor for Sorter {
         let _ = key;
         todo!()
     }
+
+    fn create_tree(&mut self, _flags: usize) -> u32 {
+        unreachable!("Why did you try to build a new tree with a sorter??? Stand up, open the door and take a walk for 30 min to come back with a better plan.");
```

Comment:
🤣

Reply:
can we also return something like this as a parse error if users make insane queries